logentries/resource_logentries_logset.go

```diff
@@ -65,7 +68,9 @@ func resourceLogentriesLogSetRead(d *schema.ResourceData, meta interface{}) erro
 		return nil
 	}
 
+	d.SetId(res.Key)
```

Comment:
This normally shouldn't be necessary in a READ method, however I see it's being added for the benefit of the data source. In this case we need to document it so that it's not mistakenly removed later by someone who doesn't know it's there for other purposes and is left wondering why a test is failing

Reply:
Added in a462da0